src/templates/blog.js

```diff
@@ -11,71 +11,79 @@ import Footer from "../sections/General/Footer";
 import { GlobalStyle } from "../sections/app.style";
 import theme from "../theme/app/themeStyles";
 
-import {graphql} from "gatsby";
+import { graphql } from "gatsby";
 import BlogList from "../sections/Blog/Blog-list";
 
-export const query = graphql`query allBlogs($skip: Int!, $limit: Int!) {
-  allMdx(
-    sort: {fields: [frontmatter___date], order: DESC}
-    filter: {fields: {collection: {eq: "blog"}}, frontmatter: {published: {eq: true}}}
-    skip: $skip
-    limit: $limit
-  ) {
-    nodes {
-      id
-      frontmatter {
-        title
-        date(formatString: "MMM Do, YYYY")
-        author
-        thumbnail {
-          childImageSharp {
-            gatsbyImageData(layout: FULL_WIDTH)
+export const query = graphql`
```

Comment:
@kayceeDev with all the reformatting included, it's difficult to tell what, if anything, has materially changed in this file? Anything?

Reply:
The only thing I changed in graphql query was to remove the limit of data returned per page so I can index all the data and make all of it searchable when a user uses the search bar. Else, our search future would be limited to the page context.